### ISSUE TITLE
Hidden app window

### ISSUE THREAD

**Issue by Maikishiti** [crashinvaders/gdx-texture-packer-gui#113]
>I'll refer to "GDX Texture Packer GUI" as "GTPG"

Some days ago I got a strange glitch in my GDX Texture Packer GUI.
 
Every time I'd open the jar (or any other GTPG executable), nothing would show on screen, but it would show on the task bar and the task manager that the app was open.

When I tried to open it by the command line (java -jar) I got a warning that read something like this (I don't have the exact words anymore).
> LibGDX 1.9.3 recommended to Vis UI version XXX, you're using LibGDX 1.9.2
> Some things might not work

So I to got the source code to change the GDX version, but it was already up-to-date and the GTPG was still not working.

Finally, I gave up on the glitch and just downloaded an older version to see if it worked, and it worked just fine. So fine indeed that the newer version returned to work out of nowhere.

Don't know how the glitch was caused, don't know exactly what fixed it.
The only clue I have to the cause of the bug is that I might have turned off my computer without closing the app. Other than that, nothing comes to my mind.

Thought I already solved it, I was recommended to post this anyway so other people would know what to do

- **TL;DR App was open but hidden, open old version and it's fixed**

**Comment by metaphore** [comment]
Hi there, thanks for the extensive description!

I'd assume you're on Windows 10?
Judging by the description, it sounds to me that the issue is with the application window location being outside the screen.

When the app is being terminated, it memorizes the exact location (position + size) to restore it on the next startup. I think your assumption is right, the powering off the computer has that side effect on the window params persistence logic. Probably by the time the app tries to save the window state, the OS has already terminated some of the windowing services and the app receives wrong values.

I'll have a look and try to reproduce it. And I have a guess that from perspective of my code, there's very little I can control in such situation. So an obvious workaround would be to validate the last saved window location on startup and reset it in case something is wrong.

In meantime to help you fix your window, you might try one of these approaches 
https://www.technipages.com/bring-off-screen-window-back-onto-screen

**Comment by metaphore** [comment]
Due to lack of new details, I guess the bug was fixed.
Feel free to reopen if the problem still stands.
